nexus-repository-p2/src/main/java/org/sonatype/nexus/repository/p2/internal/proxy/P2ProxyFacetImpl.java

```diff
@@ -238,14 +239,15 @@ protected Content doCreateOrSaveComponent(final P2Attributes p2Attributes,
     StorageTx tx = UnitOfWork.currentTx();
     Bucket bucket = tx.findBucket(getRepository());
 
-    Component component = p2DataAccess.findComponent(tx,
+    Component component = p2DataAccess.findComponentByGroupName(tx,
```

Comment:
check do we need the old method findComponent

Reply:
Removed